tests/test_configuration_set.py

```diff
@@ -101,3 +104,96 @@ def test_configurations_load_xyz():
     for config in configs:
         assert config.charge == 0
         assert config.mult == 2
+
+
+@work_in_tmp_dir()
+def test_configurations_load_xyz_with_energies_forces():
```

Comment:
This test look good, but it is very long. I suggest to split it into three parts - test of configuration, test of ConfigurationsSet and test of saving.